spec/lib/audit/catalog_to_moab_instance_spec.rb

```diff
@@ -75,8 +77,47 @@
         expect(pohandler_results).to receive(:add_result).with(
           PreservedObjectHandlerResults::ONLINE_MOAB_DOES_NOT_EXIST
         )
+        expect(pohandler_results).to receive(:add_result).with(
+          PreservedObjectHandlerResults::PC_STATUS_CHANGED, old_status: "ok", new_status: "online_moab_not_found"
+        )
         c2m.check_catalog_version
       end
+      context 'updates status correctly' do
+        before do
+          allow(Moab::StorageObject).to receive(:new).with(druid, instance_of(String)).and_return(nil)
+          # allow(pohandler_results).to receive(:add_result)
+        end
+        it 'had OK_STATUS, but is now ONLINE_MOAB_NOT_FOUND_STATUS' do
```

Comment:
what do you think about

```ruby
[
  PreservedCopy::OK_STATUS,
  ...
  PreservedCopy::VALIDITY_UNKNOWN_STATUS
].each do |orig_status
  it "changes status to ONLINE_MOAB_NOT_FOUND_STATUS from orig status: #{orig_status}" do 
    pres_copy.status = orig_status
    pres_copy.save!
    c2m.check_catalog_version
    expect(pres_copy.reload.status).to eq PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS
end
```

Reply:
oh sure, yeah, i like this approach, i think it's a fine style to use when testing lists of things that all have the same expected behavior.